src/toasts/IncomingLegacyCallToast.tsx

```diff
@@ -128,6 +129,7 @@ export default class IncomingLegacyCallToast extends React.Component<IProps, ISt
             </div>
             <AccessibleTooltipButton
                 className={silenceClass}
+                disabled={callForcedSilent}
```

Comment:
Related to the above, this would confuse me as a user, it should give the user a tooltip to explain why the app isn't letting them do this